index.md

```diff
@@ -1,7 +1,7 @@
 ---
 layout: default
-title: OLS
-description: A mentoring & training program for Open Science ambassadors
+title: A mentoring & training program for Open Science ambassadors
```

Comment:
OLS is now missing on the header and the description is too long on the top of the page

![image](https://user-images.githubusercontent.com/1842467/196104875-ee3ca95f-bfe6-4464-b8c4-56fd036c6d53.png)

Reply:
Alright @bebatut I will fix these.

Reply:
Hi @bebatut @yochannah After going through this again, I have seen that the Meta tag cannot be fixed without first restructuring the page layout. This will involve creating an appropriate tag for the hero text instead of using the page name for this. Only then can a proper meta tag and description be used without the text appearing on the site itself causing a disruption such as this.